packages/eas-cli/src/commands/fingerprint/generate.ts

```diff
@@ -0,0 +1,103 @@
+import { Flags } from '@oclif/core';
+
+import {
+  getFingerprintInfoFromLocalProjectForPlatformsAsync,
+  stringToAppPlatform,
+} from './compare';
```

Comment:
prefer to move common utilities out of the commands. IIRC oclif does a weird thing with lazy imports and commands and importing a command is relatively slow compared to a normal module. (found this while doing the `eas rollback` disambiguation command)